dev/TabView/APITests/TabViewTests.cs

```diff
@@ -167,6 +168,113 @@ static ISelectionItemProvider GetProviderFromTVI(TabViewItem item)
             }
         }
 
+        [TestMethod]
+        public void TabViewItemBackgroundTest()
+        {
+            TabView tabView = null;
+            TabViewItem tabViewItem1 = null;
+            TabViewItem tabViewItem2 = null;
+            RunOnUIThread.Execute(() =>
+            {
+                tabView = new TabView();
+
+                tabViewItem1 = CreateTabViewItem("Tab1", Symbol.Home);
+                tabViewItem2 = CreateTabViewItem("Tab2", Symbol.Document);
+
+                tabView.TabItems.Add(tabViewItem1);
+                tabView.TabItems.Add(tabViewItem2);
+
+                Content = tabView;
+            });
+
+            IdleSynchronizer.Wait();
+
+            RunOnUIThread.Execute(() =>
+            {
+                // Verify that the TabViewItem we use for background API testing here is unselected
+                Verify.IsFalse(tabViewItem2.IsSelected, "TabViewItem should have been unselected");
+
+                bool testCondition = !IsBrushUsingSolidColor(tabViewItem2.Background, Colors.Blue);
+                Verify.IsTrue(testCondition, "Default TabViewItem background color should have not been [blue]");
+
+                var tabContainer = VisualTreeUtils.FindVisualChildByName(tabViewItem2, "TabContainer") as Grid;
+
+                testCondition = !IsBrushUsingSolidColor(tabContainer.Background, Colors.Blue);
+                Verify.IsTrue(testCondition, "TabViewItem's [TabContainer] background color should have not been [blue]");
+
+                Log.Comment("Set the background color of the TabViewItem to [blue]");
+                tabViewItem2.Background = new SolidColorBrush(Colors.Blue);
+
+                testCondition = IsBrushUsingSolidColor(tabViewItem2.Background, Colors.Blue);
+                Verify.IsTrue(testCondition, "TabViewItem's background color should have been [blue]");
+
+                testCondition = IsBrushUsingSolidColor(tabContainer.Background, Colors.Blue);
+                Verify.IsTrue(testCondition, "TabViewItem's [TabContainer] background color should have been [blue]");
+
+                bool IsBrushUsingSolidColor(Brush brush, Color color)
+                {
+                    return brush is SolidColorBrush scBrush && scBrush.Color == color;
+                }
+            });
+        }
+
+        [TestMethod]
+        public void TabViewItemForegroundTest()
```

Comment:
>TabViewItemForegroundTest [](start = 20, length = 25)

These tests look similar, should we combine them?

Reply:
We could combine them but I wanted to have one test *per* API. That way, we have a clean separation of API tests and reduced complexity for each test which will also help maintaining them.

Reply:
sure, sounds reasonable.